lib/matplotlib/collections.py

```diff
@@ -1262,62 +1262,60 @@ def __init__(self,
                  **kwargs
                  ):
         """
-        *positions*
-            a sequence of numerical values or a 1D numpy array.  Can be None
+        Parameters
+        ----------
+        positions : 1D array-like object or None.
```

Comment:
Looks like the previous implementation did not support passing None as input (despite being documented that it does).  If that is correct I would fix the docstring to say it's not supported (I don't see why it should).